### PR TITLE
test(ui): Delete events chart test

### PR DESCRIPTION
**Pull request by scttcper**
This test is impossible to re-write in RTL. The test relies on accessing the components directly and testing how many times they render.

### CODE REVIEW

**Review by malwilley — COMMENTED** [review]
I wonder if we can make our chart mocks render _something_ in the test environment to make testing them possible?

Not to say we can't merge this and deal with that stuff later

File: static/app/components/charts/eventsChart.spec.jsx [hunk 1]
```diff
@@ -43,135 +42,26 @@ describe('EventsChart', function () {
...
-  });
-
-  it('re-renders if period from props changes', function () {
-    wrapper.setProps({period: '7d'});
-    wrapper.update();
-    expect(render).toHaveBeenCalledTimes(1);
-  });
-
-  it('re-renders if query from props changes', function () {
-    wrapper.setProps({query: 'newQuery'});
```
Comment (priscilawebdev) [inline static/app/components/charts/eventsChart.spec.jsx:70]: is this being reflected in the UI? if we can't target the DOM element, can we take a screenshot?

Comment (scttcper) [reply static/app/components/charts/eventsChart.spec.jsx:70]: echarts is entirely mocked out in all our tests

Comment (scttcper) [reply static/app/components/charts/eventsChart.spec.jsx:70]: maybe we can keep this test some other way, it must make a request or something

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**